pkg/monitortests/node/legacynodemonitortests/pathological_events_test.go

```diff
@@ -108,23 +113,34 @@ func Test_testBackoffStartingFailedContainer(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			e := monitorapi.Intervals{tt.interval}
 			junits := testBackoffStartingFailedContainer(e)
+
+			// Find the junit with the namespace of openshift-etcd-operator int the testname
+			var testJunits []*junitapi.JUnitTestCase
+			for _, j := range junits {
+				if strings.Contains(j.Name, namespace) {
```

Comment:
Here what I was thinking was the test would have `namespace: "openshift-etcd-operator"` and the logic would be

if strings.Contains(j.Name, tt.namespace) 

Then if the test was updated later on to test another namespace it would handle it.

But no need to hold this up for that, if / when there is a need it is easy enough to refactor.